apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *names.id
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class ActorRelatedInformationObjectsAction extends sfAction
+{
+  public function execute($request)
+  {
+    $this->response->setHttpHeader('Content-Type', 'application/json; charset=utf-8');
+
+    if ((empty($request->actorId) || !ctype_digit($request->actorId))
+      || (empty($request->page) || !ctype_digit($request->page))
+      || (isset($request->eventTypeId) && !ctype_digit($request->eventTypeId)))
+    {
+      $this->forward404();
+    }
+
+    $limit = sfConfig::get('app_hits_per_page', 10);
+    $culture = $this->context->user->getCulture();
+
+    $resultSet = self::getRelatedInformationObjects($request->actorId, $request->page, $limit, $request->eventTypeId);
```

Comment:
Not sure what do you mean with undefined, it's forwarding to a 404 page if the value of that parameter is not a digit and, if it's not a event type id, it will return no results.